getversion.sh.in

```diff
@@ -1,12 +1,13 @@
-#!/bin/sh
+#!/bin/bash
 # getversion.h will be automatically generated from this
 # Output of getversion.h goes into src/version.h
 
-githead=@SOURCE_DIR@/.git/HEAD
-version=@PACKAGE_VERSION@
+  githead=@SOURCE_DIR@/.git/HEAD
+  version=@PACKAGE_VERSION@
+
+  if [ -f $githead ]; then
+    git describe --tags --always
+  else
+    echo "$version"
+  fi
```

Comment:
Looks like whitespace change alone.

Reply:
I have already removed the trailing whitespaces. The changes on this script concern the `git describe` command which is output as it is rather than using `echo` and a subshell.